smk/binning_preparation.smk

```diff
@@ -364,13 +365,15 @@ rule map_contigs_BWA_depth_coverM:
         depth="{wd}/{omics}/8-1-binning/depth_{scaf_type}.{min_length}/batch{batch}/{illumina}.depth.txt.gz"
     shadow:
         "minimal"
-    params:
-        map_threads = config['BWA_threads'],
-        samtools_sort_threads = 3,
-        coverm_threads = lambda wildcards, threads: min(threads, 8),
     log:
         "{wd}/logs/{omics}/6-mapping/{illumina}/{illumina}.scaffolds_{scaf_type}.{min_length}.batch{batch}.bwa2.log"
+    wildcard_constraints:
+        batch='\d+',
+        illumina='[^/]+'
     resources:
```

Comment:
Could you please use `r''` (raw string) for the regex wildcard constraints?

Reply:
Done (9fab32b)